server/src/main/resources/application.conf

```diff
@@ -60,23 +60,13 @@ akka {
   jvm-exit-on-fatal-error = true
   persistence {
     journal {
-      plugin = "akka.persistence.journal.leveldb"
-      leveldb.dir = ${persistenceBaseDir}
+      plugin = "jdbc-journal"
     }
     snapshot-store {
       # Path to the snapshot store plugin to be used
-      plugin = "akka.persistence.snapshot-store.local"
-      # Local filesystem snapshot store plugin.
-      local {
-        # Class name of the plugin.
-        class = "akka.persistence.snapshot.local.LocalSnapshotStore"
-        # Dispatcher for the plugin actor.
-        plugin-dispatcher = "akka.persistence.dispatchers.default-plugin-dispatcher"
-        # Dispatcher for streaming snapshot IO.
-        stream-dispatcher = "akka.persistence.dispatchers.default-stream-dispatcher"
-        # Storage location of snapshot files.
-        dir = ${persistenceBaseDir}"/snapshots"
-      }
+      plugin = "jdbc-snapshot-store"
+      // Enable the line below to automatically start the snapshot-store when the actorsystem is started
```

Comment:
remove all changes in this file.

Reply:
We should be using this config in ACP: https://github.com/UKHomeOffice/drt-v2/blob/master/server/src/main/resources/postgres.conf

Which you can get by specifying this java cli arg:
```
-Dconfig.resource=prod.conf
```